x-pack/plugins/enterprise_search/public/applications/app_search/components/relevance_tuning/components/precision_slider/precision_slider.test.tsx

```diff
@@ -82,7 +84,7 @@ describe('PrecisionSlider', () => {
   it('contains a documentation link', () => {
     const documentationLink = wrapper.find('[data-test-subj="documentationLink"]');
 
-    expect(documentationLink.prop('href')).toContain('/precision-tuning.html');
+    expect(documentationLink.prop('href')).toContain(`${docLinks.appSearchPrecision}`);
```

Comment:
```suggestion
    expect(documentationLink.prop('href')).toContain(docLinks.appSearchPrecision);
```
No need for a template literal here